src/Castle.Facilities.AspNetCore/Castle.Facilities.AspNetCore.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+	</PropertyGroup>
+
+	<Import Project="..\..\buildscripts\common.props"></Import>
+
+	<PropertyGroup>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+		<PackageId>Castle.Facilities.AspNetCore</PackageId>
+		<Title>Castle Windsor AspNet Core facility</Title>
+		<Description>Castle Windsor AspNet Core facility lets you easily add windsor to aspnet core apps.</Description>
+		<PackageTags>castle, windsor, inversionOfControl, DependencyInjection, aspnet, core</PackageTags>
+		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+		<AssemblyName>Castle.Facilities.AspNetCore</AssemblyName>
+		<RootNamespace>Castle.Facilities.AspNetCore</RootNamespace>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.AspNetCore" Version="2.0.1" />
+		<PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.0.2" />
+		<PackageReference Include="Microsoft.AspNetCore.Mvc.Razor" Version="2.0.2" />
```

Comment:
I'm surprised this facility needs ASP.NET Core MVC and Razor, and not just the main ASP.NET Core package. I haven't had a chance to check out the work the Nancy guys have done to get Nancy running on ASP.NET Core but I thought this facility might be a drop in.

Reply:
Razor is there because of the activator for [TagHelpers](https://github.com/fir3pho3nixx/Windsor.AspNetCore/blob/master/Castle.Facilities.AspNetCore/Activators/DelegatingTagHelperActivator.cs#L19).

Mvc is there because of the activator for [Controllers](https://github.com/fir3pho3nixx/Windsor.AspNetCore/blob/master/Castle.Facilities.AspNetCore/Activators/DelegatingControllerActivator.cs#L19).

Open to any idea's that could make this easier.

Reply:
> Open to any idea's that could make this easier.

I've got no ideas, I think just leave this as is for now. Nancy is different with its container usage anyway wired through the bootstrapper so let's get this going and the first person to want Windsor on Nancy can sort that one out.